app/views/dashboard/_dashboard_ect.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <div class="blue-header">
+      <h1 class="govuk-heading-xl blue-header__title">Welcome <%= current_user.name %></h1>
+      <strong><%= govuk_link_to "Edit name", edit_preferred_name_path, class: "blue-header__text", "data-test": "edit-name" %></strong>
+    </div>
+
+    <% if current_user.early_career_teacher_profile.core_induction_programme %>
+      <%= govuk_link_to "Go to your module materials",
+        cip_path(current_user.early_career_teacher_profile.core_induction_programme),
+        button: true %>
+    <% end %>
```

Comment:
Pretty sure it will break for admins

Reply:
Cause they don't have a ect profile, and so nil has no method 😢 

If you add a default dashboard and only show this one for ects it should be good.